cloudinit/distros/__init__.py

```diff
@@ -1248,6 +1248,13 @@ def build_dhclient_cmd(
             "/bin/true",
         ] + (["-cf", config_file, interface] if config_file else [interface])
 
+    @staticmethod
+    def eject_media(device: str) -> None:
+        cmd = ["/lib/udev/cdrom_id", "--eject-media", device]
+        if uses_systemd():
```

Comment:
Wait...did I do this backwards? Don't we want to use `lib/udev/cdrom_id` if we're using systemd?

Reply:
No that was me fighting with with tox (it didn't like your one-line conditional).  Let me fix it now.